CI/install-dependencies-macos.sh

```diff
@@ -7,22 +7,26 @@ brew install ffmpeg
 brew install libav
 
 # qtwebsockets deps
-brew install qt5
+# qt latest
+#brew install qt5
+
+# qt 1.9.2
```

Comment:
super minor nit: `5.9.2`

Reply:
Argh my friend the typo... Should be fixed by now.